roles/docker/tasks/main.yml

```diff
@@ -67,7 +67,7 @@
     dest: /etc/sysconfig/docker
     regexp: '^{{ item.reg_conf_var }}=.*$'
     line: "{{ item.reg_conf_var }}='{{ item.reg_fact_val | oo_prepend_strings_in_list(item.reg_flag ~ ' ') | join(' ') }}'"
-  when: item.reg_fact_val != '' and  docker_check.stat.isreg
+  when: item.reg_fact_val != '' and  (docker_check.stat.isreg is define and docker_check.stat.isreg)
```

Comment:
We may want to check for the existence of the `is_reg` key here rather than using `is defined`.

```
when: item.reg_fact_val != '' and  ('is_reg' in docker_check.stat and docker_check.stat.isreg)
```

Side note: Where did you get your docker package? Assuming it came from centos/rhel/fedora it should create `/etc/sysconfig/docker`.